docs/tutorials/grover_with_sampler.ipynb

```diff
@@ -7,17 +7,17 @@
    "source": [
```

Comment:
<p>The way you've done the "admire mildly what the user has achieved" and the summary works particularly well for this specific tutorial in part because of the "looking for phone number" context you added. However, in other tutorials this parts could easily become a bit repetitive.</p><p> </p><p>Let's say the tutorial is the getting started with Qiskit one. You could say for example "Congratulations! You have created your first quantum circuit!" and then the summary (just after this) would read something like "In this tutorial you have created, run and visualized the results of a quantum circuit". May be lack of inspiration on my part of course!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-ibm-runtime/pull/393/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>Yes I agree with you that the summary section and the lasts part of the tutorial often repeat each other. The summary also repeats the similar content in the overview. As a general rule I would say we can rephrase them slightly different from each other. But in specific cases we should make the decisions depending on the context.</p>